src/common/components/clusterConfiguration/DiskEncryptionFields/DiskEncryptionControlGroup.tsx

```diff
@@ -68,21 +69,22 @@ const DiskEncryptionControlGroup: React.FC<DiskEncryptionControlGroupProps> = ({
       setFieldValue('enableDiskEncryptionOnWorkers', false);
     }
   }, [isSNO, setFieldValue]);
-
-  const disableMessage = 'This option is not editable after the draft cluster is created';
+  const { t } = useTranslation();
+  const disableMessage = t('ai:This option is not editable after the draft cluster is created');
   const tooltipProps = {
     hidden: !isDisabled,
     content: disableMessage,
   };
+
   return (
     <Stack hasGutter>
       <StackItem>
         <SwitchField
           tooltipProps={tooltipProps}
           name="enableDiskEncryptionOnMasters"
-          label={`Enable encryption of installation disk${
-            !isSNO ? 's on control plane nodes' : ''
-          }`}
+          label={t('ai:Enable encryption of installation disk {{control_text}}', {
+            control_text: !isSNO ? t("ai:'s on control plane nodes") : '',
+          })}
```

Comment:
`control_text` and similar props should be used for values that are not being translated which is not this case.

```suggestion
          label={isSNO ? t('ai:Enable encryption of installation disk') : t('ai:Enable encryption of installation disk's on control plane nodes')}
```